bucket/vscodium.json

```diff
@@ -50,6 +50,11 @@
         "if (!(Test-Path \"$dir\\data\\user-data\") -and (Test-Path \"$env:AppData\\VSCodium\")) {",
         "    info '[Portable Mode] Copying user data...'",
         "    Copy-Item \"$env:AppData\\VSCodium\" \"$dir\\data\\user-data\" -Recurse",
+        "}",
+        "$extensions_file = \"$dir\\data\\extensions\\extensions.json\"",
```

Comment:
Do the same here for this line as well.

Reply:
I don't recommend that. As I said in the PR submission, any new extension will use the path with the version number in it (`$dir`). If we replace the path with `$persist_dir`,  we will end up having extensions present at the time of the update with `$persist_dir` and the new ones with `$dir`. This may create confusion and alter the behaviour of vscode in unexpected ways (sync, ...). Changes in vscode seem to be coming from [this bug](https://github.com/microsoft/vscode/issues/166147) and this [PR](https://github.com/microsoft/vscode/pull/166648). As it touches a bunch of files, I think it's better to keep all files consistent.

Reply:
Oh! You're right, I didn't think about how it will create bugs for vscode.